composer.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "beans/beans-child",
+  "description": "Beans theme starter child theme.",
+  "type": "wordpress-theme",
+  "license": "GPL-2.0+",
+  "homepage": "http://www.getbeans.io/",
+  "support": {
+    "issues": "https://github.com/Getbeans/Beans-Starter-Child-Theme/issues",
+    "source": "https://github.com/Getbeans/Beans-Starter-Child-Theme"
+  },
+  "prefer-stable": true,
+  "repositories": [
+    {
+      "type": "package",
+      "package": {
+      "name": "xwp/wp-dev-lib",
+      "version": "1.0.1",
+        "source": {
+          "url": "https://github.com/xwp/wp-dev-lib.git",
+          "type": "git",
+          "reference": "master"
+        }
+      }
+    }
+  ],
+  "require-dev": {
```

Comment:
I'd suggest that we set up this `composer.json` with the same dependencies and scripts as in Beans.  It would then give a developer the environment, s/he needs to start building a theme and writing tests.

We could think of the child theme as the starting _development environment for a new theme_.